src/app/routes/query.py

```diff
@@ -69,24 +72,65 @@ def route_results():
             '--query-name', f'{query_name}',
             '--auths', ",".join(selected_auths),
             f'-f {data_type}' if data_type else '',
-            f'{"-d" if decode_raw_data else ""}',
-            '-o', output_location
+            f'{"-d" if decode_raw_data else ""}'
```

Comment:
I don't think this is needed anymore. Since we're not doing anything with the decoded data and are just serving the binary back as decoded files.